### ISSUE TITLE
GeometryType not handled properly

### ISSUE THREAD

**Issue by chrisnatali** [SEL-Columbia/networker#85]
Getting ImportError: GeometryType -2147483646 not supported when running with this existing network:  

[Reseau_MT_FINAL-WGS84_dec_degrees.zip](https://github.com/SEL-Columbia/networker/files/249087/Reseau_MT_FINAL-WGS84_dec_degrees.zip)

zip of run that's failing:  
http://45.55.201.87:8000/data/e3c7d5d0-a7db-4ad4-9ea7-eeb16637abae/input.zip

**Comment by vr2262** [comment]
Might help?

http://osgeo-org.1560.x6.nabble.com/Clip-unsupported-geometrytype-2147483646-td5261440.html

**Comment by chrisnatali** [comment]
Definitely.  @Naigege @sherpashaky may want to look at the link above in case there's something they can do with QGIS or JOSM to remedy this.

**Comment by Naigege** [comment]
Fixed it by copying all features in target layer, pasting in one new temporary layer and saving as a new shape file. 

It seems the issue has another explaination of 2.5D geometry type. QGIS is working on here:
https://github.com/qgis/QGIS/commit/59522582bfd964da755ebee2086b432f49fe3cdf

**Comment by chrisnatali** [comment]
Cool, tabling this issue for now then.  

FYI we only handle the 'wkbLineString' and 'wkbMultiLineString' from all of the wkb types.  

In case it helps, these are the wkb types/codes available in ogr:  
[('wkb25Bit', -2147483648),
 ('wkb25DBit', -2147483648),
 ('wkbCircularString', 8),
 ('wkbCircularStringZ', 1008),
 ('wkbCompoundCurve', 9),
 ('wkbCompoundCurveZ', 1009),
 ('wkbCurvePolygon', 10),
 ('wkbCurvePolygonZ', 1010),
 ('wkbGeometryCollection', 7),
 ('wkbGeometryCollection25D', -2147483641),
 ('wkbLineString', 2),
 ('wkbLineString25D', -2147483646),
 ('wkbLinearRing', 101),
 ('wkbMultiCurve', 11),
 ('wkbMultiCurveZ', 1011),
 ('wkbMultiLineString', 5),
 ('wkbMultiLineString25D', -2147483643),
 ('wkbMultiPoint', 4),
 ('wkbMultiPoint25D', -2147483644),
 ('wkbMultiPolygon', 6),
 ('wkbMultiPolygon25D', -2147483642),
 ('wkbMultiSurface', 12),
 ('wkbMultiSurfaceZ', 1012),
 ('wkbNDR', 1),
 ('wkbNone', 100),
 ('wkbPoint', 1),
 ('wkbPoint25D', -2147483647),
 ('wkbPolygon', 3),
 ('wkbPolygon25D', -2147483645),
 ('wkbUnknown', 0),
 ('wkbXDR', 0)]
